tests/service-mocks/media-player-mpris-mock/CMakeLists.txt

```diff
@@ -8,10 +8,6 @@ find_package(Qt5DBus REQUIRED)
 include_directories(${Qt5DBus_INCLUDE_DIRS}
                     "${CMAKE_SOURCE_DIR}/tests/service-mocks")
 
-if(EXISTS "/usr/share/accountsservice/interfaces/com.lomiri.AccountsService.SecurityPrivacy.xml")
-    add_definitions(-DHAS_LOMIRI_ACCTSERVICE_SOUND_SETTINGS)
-endif()
```

Comment:
The above seems to be about SecurityPrivacy. Not sound. Maybe an old mismatch. But maybe not candidate for being dropped here. Rather a fix in another PR might be required?

Reply:
Ignoring, as we agreed to simplify this in the next PR.